Src/WitsmlExplorer.Frontend/components/ContentViews/ServerManager.tsx

```diff
@@ -92,8 +92,8 @@ const ServerManager = (): React.ReactElement => {
 
   const updateVisibleObjects = (supportedObjects: string[]) => {
     const updatedVisibility = { ...allVisibleObjects };
-    const hiddenItems = localStorage.getItem(STORAGE_FILTER_HIDDENOBJECTS_KEY)?.split(",") || [];
-    hiddenItems.forEach((objectType) => (updatedVisibility[objectType as ObjectType] = VisibilityStatus.Hidden));
+    const hiddenItems = getLocalStorageItem(STORAGE_FILTER_HIDDENOBJECTS_KEY) || [];
```

Comment:
Is it possible to use the `useLocalStorageState` on the top level of this component and use the state provided by the hook instead?

Reply:
Using the hook would only be an extra step towards calling the getLocalStorageItem as the value is not updated in ServerManager. It would also not catch changes made while ServerManager is open, so additional logic would have to be added to fix that. I think it's fine to leave it as is since using the hook adds no benefit in this case.